src/main/java/com/revature/model/Role.java

```diff
@@ -1,9 +1,87 @@
 package com.revature.model;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "Role")
 public class Role {
-	
+
```

Comment:
Trainer doesn't need a role. It won't break anything, but it's an unnecessary foreign key. Let me know if you want to keep it for any reason.

Reply:
Ill take it out